src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -74,7 +74,7 @@ object SeedJointSelector {
     } else if (jointSets.head(0).shape.isEmpty) {
       val (seedJoints, leftOverJoints) = searchJointSets(jointSets, inputVolume, numPartitions, jointSetSpan)
       val partitioning = seedJoints.foldLeft(Seq(inputVolume)) { (currentBlocks, joint) =>
-        currentBlocks.flatMap(_.cut(joint))
+        currentBlocks.flatMap(_.cut(joint, minSize = 0.0, maxAspectRatio = Double.PositiveInfinity))
```

Comment:
I think the reason we decided not to go with it was because we thought we'd have to bind one of the arguments and do some overly fancy stuff.

Looking at it now, your suggestions seems like a good idea. I'll change it.